single_tech_samples/purview/sample2_data_plane_permissions_release_mng/README.md

```diff
@@ -0,0 +1,65 @@
+# Github Action to Set Purview Permissions
+
+A github action to manage permissions with purview.
+
+## Problem Statement
+
+Managing purview permissions cannot be easily done programmatically because permissions are managed in the data plane and not in IAM. This makes it difficult to assign permissions using IaCs such as terraform or bicep. Also, the only permission currently available to assign via `az cli` is the root collection admin.
+
+This action allows you to assign any purview permission(s) to a user or security group.
+
+## Sample Usage
+
+> *Note: It is preferable to clone or fork this action when testing to avoid populating the actions log for this repo.*
+
+To use the action in a workflow with a service principal for example, you can [create a service principal](https://learn.microsoft.com/en-us/azure/developer/github/connect-from-azure?tabs=azure-portal%2Clinux#create-a-service-principal), connect to github and ensure right permissions to successfully run the workflow.
+
```

Comment:
Please mention that this $GITHUB_ACTION_PATH variable needs to be created upfront and how (https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-a-repository)